gcc/config/arc64/arc64.h

```diff
@@ -302,7 +302,7 @@ enum reg_class
 {								\
   { 0x00000000, 0x00000000, 0x00000000, 0x00000000 }, /* NO_REGS */	\
   { 0x0000f00f, 0x00000000, 0x00000000, 0x00000000 }, /* AC16_REGS */	\
-  { 0x00001fff, 0x00000000, 0x00000000, 0x00000000 }, /* SIBCALL_REGS */ \
+  { 0x00003fff, 0x00000000, 0x00000000, 0x00000000 }, /* SIBCALL_REGS */ \
```

Comment:
This change isn't reflected in the commit message and isn't very obvious without an explanation.

Reply:
I now refer in the commit message about the R13